### PR TITLE
For #161 CI fixed

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #161 corrected puzzle and added jacoco.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#163](https://github.com/artipie/rpm-adapter/issues/163) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/ec72d07d-237a-4d5e-abc4-b0d8f73e7f5f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/rpm/meta/XmlRepomd.java [hunk 1]
```diff
@@ -74,17 +80,18 @@
      * @param path Temporary file path
      */
     public XmlRepomd(final Path path) {
-        this(XmlRepomd.xmlStreamWriter(path), path);
+        this(path, XmlRepomd.outputStream(path));
```
Comment (Vatavuk) [inline src/main/java/com/artipie/rpm/meta/XmlRepomd.java:83]: @olenagerasimova then will be `this(new XmlFile(path))`

File: src/main/java/com/artipie/rpm/meta/XmlRepomd.java [hunk 2]
```diff
@@ -126,25 +133,37 @@ public void close() throws IOException {
...
+        }
+    }
+
+    /**
+     * New XML stream writer from path.
+     * @param stream Output steam
      * @return XML stream writer
      */
-    private static XMLStreamWriter xmlStreamWriter(final Path path) {
+    private static XMLStreamWriter xmlStreamWriter(final OutputStream stream) {
```
Comment (Vatavuk) [inline src/main/java/com/artipie/rpm/meta/XmlRepomd.java:160]: @olenagerasimova this method and previous method `outputStream` will no longer be needed

**Comment by olenagerasimova** [comment]
@Vatavuk thanks, corrected

**Comment by 0crat** [comment]
Job was finished in 20 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Pull request merged**